### PR TITLE
fix: define heading and color styles in scss to make it easier to use…

### PR DESCRIPTION
**Pull request by jasonzhwang**
Define headings
image

Define colors
image
image

so that 

developer can directly use heading styles by referring className, and define their own color styles in font or background-color if needed.

### CODE REVIEW

File: src/components/DesignSytem/DesignSystem.module.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+.designContainer {
+    display: flex;
+    gap: 100px;
+  }
+
+.headingColumn{
+  display: flex;
+  flex-direction: column;
+}
```
Comment (tatoMa) [inline src/components/DesignSytem/DesignSystem.module.scss:28]: Do you think these should be inside of the design system (theme style) or they are only for specific pages?

File: src/pages/designsystem.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import DesignSystem from '../components/DesignSytem';
```
Comment (Double-Twelve) [inline src/pages/designsystem.tsx:2]: 1. spell error should `DesignSystem`
2. this file's name should `design-system`

File: src/components/DesignSytem/DesignSystem.module.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+$greyIcon: #bababa;
+$greyDark: #757575;
+
+/* Secondary Color*/
+$cyan: #5996a5;
+$green: #639b6d;
+$pink: #a15993;
+$red: #a95151;
+$yellow: #c4a24c;
+$orange: #cb5b43;
```
Comment (Double-Twelve) [inline src/components/DesignSytem/DesignSystem.module.scss:18]: these variables should be extracted to a file for later use

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,3 +1,26 @@
+import Head from 'next/head';
+
 export default function Home() {
-  return <div>Hello, this is TaoTalk </div>;
+  return (
+    <>
+      <Head>
+        <title>TaoTalk</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (tatoMa) [inline src/pages/index.tsx:8]: Move this tag into _app file. This tag will always be the same with every single page.

File: src/pages/index.tsx [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+        <meta name="description" content="TaoTalk" />
+        <link rel="icon" href="/favicon.ico" />
```
Comment (tatoMa) [inline src/pages/index.tsx:10]: Move this tag into _app file. This tag will always be the same with every single page.

File: src/styles/color.module.scss
Comment (tatoMa) [inline src/styles/color.module.scss:1]: Why is this file still a module CSS?

**Review by PengchengYuGithub — APPROVED** [review]
LGTM

**Pull request merged**